pyproject.toml

```diff
@@ -24,11 +24,12 @@ authors = [
     {name = "Lyz", email = "lyz-code-security-advisories@riseup.net"},
 ]
 license = {text = "GPL-3.0-only"}
-requires-python = ">=3.8"
+requires-python = ">=3.9.1"
 dependencies = [
     "click>=8.1.3",
     "ruyaml>=0.91.0",
-    "maison>=1.4.0",
+    "maison>=2.0.0",
+    "pydantic>=2.8.2",
```

Comment:
`pydantic` is not a `maison` dependency anymore (see [release notes](https://github.com/dbatten5/maison/releases/tag/v2.0.0#:~:text=The%20use%20of%20pydantic%20to%20perform%20validation%20against%20a%20schema%20is%20still%20preferred%2C%20but%20it%27s%20now%20up%20to%20the%20user%20to%20include%20this%20dependency.)) 
It is used optionally for validation, which we use.